src/main/java/net/dv8tion/jda/internal/requests/restaction/pagination/PaginationActionImpl.java

```diff
@@ -347,6 +347,12 @@ public void forEachRemaining(@Nonnull final Procedure<? super T> action)
         }
     }
 
+    @Nonnull
+    protected String getLastKeyValue(long last)
```

Comment:
I feel like `formatKey` would be a better name for this method, because the initial forwards order might break, if we just pass a simple `"0"` instead of a proper format for given pagination endpoints relying on different formats such as ISO8601.